app/helpers/location_helper.rb

```diff
@@ -120,4 +120,18 @@ def self.filter_by_name(samples_with_metadata, query)
       samples.where("`metadata`.`string_validated_value` IN (?)", query)
     end
   end
+
+  # Normalize some location names from the provider for matching consistency.
+  # Ex: Treat "United States of America" and "USA" as the same in cases where
+  # the provider is not internally consistent.
+  #
+  # See config/initializers/location_name_aliases.rb and add important known
+  # aliases there.
+  def self.normalize_name_aliases(name)
+    if LOCATION_NAME_ALIASES && LOCATION_NAME_ALIASES.key?(name)
```

Comment:
there is a chance that we could want two same-named entities to have different aliases. for example, washington the state and washington the city.

Reply:
That's a great point. Adapted to split by geo_levels 👍